### PR TITLE
secboot: helpers for dealing with PCR handles and TPM resources

### PR DESCRIPTION
**Pull request by bboozzoo**
Two more helpers for factory reset. One reads the PCR handle embedded in the sealed key object (for handle rotation). The other releases resources in TPM's non volatile memory associated with given PCR handles.

### CODE REVIEW

File: secboot/secboot_sb_test.go [hunk 1]
```diff
@@ -1890,3 +1890,67 @@ func (s *secbootSuite) TestUnlockVolumeUsingSealedKeyIfEncryptedFdeRevealKeyBadJ
...
 }
+
+func (s *secbootSuite) TestPCRHandleOfSealedKey(c *C) {
+	d := c.MkDir()
+	h, err := secboot.PCRHandleOfSealedKey(filepath.Join(d, "not-found"))
+	c.Assert(err, ErrorMatches, "cannot open key file: .*/not-found: no such file or directory")
+	c.Assert(h, Equals, uint32(0))
+
+	skf := filepath.Join(d, "sealed-key")
+	// partially valid salealed key with correct header magic
```
Comment (mardy) [inline secboot/secboot_sb_test.go:1901]:
```suggestion
	// partially valid sealed key with correct header magic
```

**Review by mardy — APPROVED** [review]
LGTM, thanks!

File: secboot/secboot_tpm.go [hunk 2]
```diff
@@ -559,3 +560,52 @@ func efiImageFromBootFile(b *bootloader.BootFile) (sb_efi.Image, error) {
...
+	tpm, err := sbConnectToDefaultTPM()
+	if err != nil {
+		err = fmt.Errorf("cannot connect to TPM device: %v", err)
+		return err
+	}
+	defer tpm.Close()
+
+	for _, handle := range handles {
+		logger.Debugf("releasing PCR handle %#x", handle)
+		if err := tpmReleaseResources(tpm, tpm2.Handle(handle)); err != nil {
```
[the remark this answers is not in the document]
Comment (bboozzoo) [reply secboot/secboot_tpm.go:611]: updated to try all the handles

**Review by pedronis — APPROVED** [review]
thanks

**Pull request merged**